series.tex

```diff
@@ -0,0 +1,93 @@
+% Series expansion (Differentiate between the two approaches being used)
+\subsubsection{Series Expansion}
+
+SymPy is able to calculate the symbolic series expansion of an arbitrary series
+or expression involving elementary and special functions and multiple
+variables. For this it has two different implementations- the \texttt{series}
+method and Ring Series.
+
+The first approach stores a series as an object of the \texttt{Basic} class.
+Each function has its specific implementation of its expansion which is able to
+evaluate the Puiseux series expansion about a specified point. For example,
+consider a Taylor expansion about 0:
+
+\begin{verbatim}
+>>> from sympy import symbols, series
+>>> x, y = symbols('x, y')
+>>> series(sin(x+y) + cos(x*y), x, 0, 2)
+1 + sin(y) + x*cos(y) + O(x**2)
+\end{verbatim}
+
+The newer and much faster\cite{sympyRingSeries} approach called Ring Series makes use of the
```

Comment:
@scopatz I have cited the SymPy here. Will update the doc with more speed comparisons.